app/controllers/api/v1/events_controller.rb

```diff
@@ -25,54 +23,54 @@ def group_events
     end
 
     def add_movie
-      event = Event.find(params[:event_id])
+      event_1 = Event.find(params[:event_id])
```

Comment:
is it possible to use `params[:event][:id]` structure so can reuse `event_params`